sorbet/rbi/gems/yard-sorbet@0.6.1.rbi

```diff
@@ -189,7 +189,9 @@ end
 YARDSorbet::NodeUtils::ATTRIBUTE_METHODS = T.let(T.unsafe(nil), Array)
 
 # Node types that can have type signatures
+# rubocop:disable Sorbet/TypeAliasName
```

Comment:
This is a generated file we shouldn't change it manually.

Let's exclude `sorbet/rbi/gems` from the cop instead.